src/logic/nfts/collections.ts

```diff
@@ -251,6 +251,7 @@ export function getCollectionsExtraWhere(options: NFTFilters) {
 export function collectionsShouldFetch(filters: NFTFilters) {
   if (
     filters.isLand ||
+    filters.status !== undefined ||
```

Comment:
Should we add `filters.isOnRent` here? 
It shouldn't fetch the collections subgraph if the `isOnRent` filter is present, right?

Reply:
I forgot, but we should remove the `filters.status` from here because the collections only retrieve wearables, not LAND, that's why the `isOnRent` filter is not set here. I'll remove it!